packages/fx-core/tests/plugins/resource/localdebug/unit/index.test.ts

```diff
@@ -366,10 +373,11 @@ describe(LocalDebugPluginInfo.pluginName, () => {
     });
 
     it("cli", async () => {
-      pluginContext.platform = Platform.CLI;
+      // pluginContext.platform = Platform.CLI;
```

Comment:
This test case is for CLI platform, so CLI is needed.

Reply:
ok